public/app/features/trails/banners/NativeHistogramBanner.tsx

```diff
@@ -0,0 +1,231 @@
+import { css } from '@emotion/css';
+import { useState } from 'react';
+
+import { GrafanaTheme2 } from '@grafana/data';
+import { useStyles2, useTheme, Alert, Button } from '@grafana/ui';
```

Comment:
`useTheme` is deprecated

```suggestion
import { useStyles2, useTheme2, Alert, Button } from '@grafana/ui';
```